tests/foreman/api/test_activationkey.py

```diff
@@ -281,6 +338,21 @@ def test_update_auto_attach(self):
         ).update(['auto_attach'])
         self.assertNotEqual(act_key.auto_attach, act_key_2.auto_attach)
 
+    def test_positive_delete_1(self):
+        """@Test: Create activation key and then delete it.
+
+        @Assert: Activation key is successfully deleted.
+
+        @Feature: ActivationKey
+
```

Comment:
No need to add this blank line.

For more info check: https://www.python.org/dev/peps/pep-0008/#documentation-strings

PS: I know that we have been doing this for a long time, but some time ago I read again that section and figured out that we ensured a non PEP8 standard.